pandas/tests/arrays/sparse/test_array.py

```diff
@@ -1295,3 +1295,14 @@ def test_map_missing():
 
     result = arr.map({0: 10, 1: 11})
     tm.assert_sp_array_equal(result, expected)
+
+
+def test_dropna_sparse_column_doesnt_drop_nonna():
+    # GH-28287
+    arr = SparseArray([np.nan, 1])
```

Comment:
```suggestion
    arr = SparseArray([np.nan, 1], fill_value=fill_value)
```

Could you parametrize this test over a fill_value of np.nan and 1?

Reply:
Just to check, the expected behaviour would be the same in case fill_value=1, correct?

Reply:
Yes, the expected result is the same. But I want to ensure that it works both in cases where the "missing value" is the fill value and where it is not the fill value.